src/components/MessageDataContent.tsx

```diff
@@ -74,15 +104,16 @@ const DataContainer = styled.div`
   justify-content: flex-start;
   align-items: flex-start;
   gap: 4px;
-  width: 100%;
+  width: calc(100% - 28px);
```

Comment:
`28px` for what?

Reply:
`width: 100%` -> `flex: 1`

Reply:
hmm let me try

Reply:
No it breaks. 28 is bar + margin
<img width="552" alt="Screenshot 2024-05-10 at 11 24 48 AM" src="https://github.com/sendbird/chat-ai-widget/assets/16806397/4f5263f8-c7de-4f05-9b8b-51f58e9cb69f">
<img width="435" alt="Screenshot 2024-05-10 at 11 24 01 AM" src="https://github.com/sendbird/chat-ai-widget/assets/16806397/6790552f-5878-4d12-8a7b-1350aaf8d5e6">

Reply:
Hmm.. how did you set the style?

## flex:1
![image](https://github.com/sendbird/chat-ai-widget/assets/26326015/1bc391b7-880b-4a9a-836e-53b8e2d34943)

## width: calc(100% - 28px);
![image](https://github.com/sendbird/chat-ai-widget/assets/26326015/55b4dd24-6e7d-46de-a8fe-f42c87fa8597)

### diff
<img width="436" alt="image" src="https://github.com/sendbird/chat-ai-widget/assets/26326015/3a53b7b5-3309-42f5-8bec-20e9b1039856">

Reply:
![Screenshot 2024-05-10 at 1 32 59 PM](https://github.com/sendbird/chat-ai-widget/assets/16806397/c988b055-37a2-44f2-9689-e3036aedbd45)

Reply:
Could you add `word-break: break-word;` style to DataContainer?

Reply:
oh nice!!! now flex:1 works!

Reply:
actually dont even need flex:1

Reply:
Even without 'flex: 1', does the SideBar width still appear as 4px ?

Reply:
wow you are right. without it, width is not 4px (smaller). i dont understand 4px is specified but why reduced?